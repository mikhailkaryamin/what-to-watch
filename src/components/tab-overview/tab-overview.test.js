import React from 'react';
import renderer from 'react-test-renderer';

import TabOverview from './tab-overview.jsx';


describe(`Tab overview`, () => {
  const DESCRIPTION = `simple text`;
  const DIRECTOR = `big boss`;
  const RATING = 9;
  const SCORE_COUNT = 100;
  const STARRING = [
    `Mike`,
    `Sam`,
    `John`,
  ];
  const wrapper = renderer
    .create(<TabOverview
      description={DESCRIPTION}
      director={DIRECTOR}
      rating={RATING}
      scoreCount={SCORE_COUNT}
      starring={STARRING}
    />);

  test(`should render tab overview`, () => {
    expect(wrapper).toMatchSnapshot();
  });
});
