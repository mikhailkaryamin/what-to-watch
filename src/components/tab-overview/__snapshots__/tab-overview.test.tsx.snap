// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tab overview should render tab overview 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      9
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Very good
      </span>
      <span
        className="movie-rating__count"
      >
        100
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    simple text
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        big boss
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        Mike
        Sam
        John
      </strong>
    </p>
  </div>,
]
`;
